src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -159,6 +168,41 @@ export default {
                 this.changeFeatureTitle({ feature: this.feature, title: value })
             },
         },
+        geometry() {
+            /*
+                Openlayers polygons coordinates are in a triple array
+                The first array is the "ring", the second is to hold the coordinates, which are in an array themselves
+                We don't have rings in this case, so we need to create an ol geometry
+                */
+
+            const geom = [this.feature.coordinates]
+
+            return new Polygon(geom)
+        },
+        isClosed() {
+            return (
+                this.feature.coordinates.length > 3 &&
+                this.feature.coordinates[0][0] ===
+                    this.feature.coordinates[this.feature.coordinates.length - 1][0] &&
+                this.feature.coordinates[0][1] ===
+                    this.feature.coordinates[this.feature.coordinates.length - 1][1]
+            )
+        },
+        length() {
+            const length = getLength(this.geometry)
+
+            if (length > 100) {
+                return `${round(length / 1000)} km`
+            }
+            return `${round(length)} m`
+        },
+        area() {
+            const area = getArea(this.geometry)
+            if (area > 10000) {
+                return `${round(area / 1000000)} km`
+            }
+            return `${round(area)} m`
+        },
```

Comment:
As you are already writing the `2` of `km2` and `m2` directly in the template, it might be better to move everything unit related there. Here it is a bit confusing to have a `area` props return a `m` or `km` number, it is the wrong unit.
So maybe it would be good to have `m` and `m2` in the template, and you only write the `k` whenever you reduce the size of the number by the corresponding factor. Or do something like

```
length() {
    return getLength(this.geometry)
}
lengthRounded() {
    if (this.length > 100) {
        return round(length / 1000.0)
    }
    return round(length)
}
// same for area
```
and in the template have something like
```
<div v-if="isFeatureLine">
    <font-awesome-icon :icon="['far', 'square']" /> {{ lengthRounded }}{{ length > 100 ? 'km' : 'm' }}
</div>
```

And don't forget that, in JS, if you divide by an integer number, the result of the division will be an integer too. If you want a float number result, you have to divide by a floating number (so you have to write the `.0` to the divisor)